website/static/manifest.json

```diff
@@ -4,7 +4,7 @@
   "theme_color": "#2196f3",
   "background_color": "#424242",
   "display": "standalone",
-  "scope": "",
+  "scope": "/",
```

Comment:
Fixes this error:

> Manifest: property 'scope' ignored. Start url should be within scope of scope URL.

Reply:
This also breaks the PWA installation on the preview ^^

https://deploy-preview-3587--docusaurus-2.netlify.app/classic/

As the scope is relative to manifest, I guess "" means "/classic/manifest.json".

Maybe "./" would work, as it would lead to "/classic/" ? worth trying

Reply:
> Maybe "./" would work

Did this, can you confirm that it worked?